hw/ip/pwrmgr/data/pwrmgr.hjson

```diff
@@ -12,7 +12,7 @@
     {clock: "clk_i", reset: "rst_ni", primary: true},
     {reset: "rst_main_ni"},
     {clock: "clk_slow_i", reset: "rst_slow_ni"}
-    {clock: "clk_esc_i", reset: "rst_esc_ni"}
```

Comment:
we might need to keep `clk_esc_i`.  There is no guarantee that the escalation module is definitely going to be the same as `clk_lc`.  If recall, I thought the escalation receiver output was treated as async, can you check?

Reply:
escalation modules are on clkmgr_aon_clocks.clk_io_div4_secure where clk_lc also comes.  
As I checked, escalation receiver output is also treated as sync without any synchronization.
If my observation is different from what you recall, let me know. 

For naming, do you want to use clk_esc_i/rst_esc_ni for pwrmgr fault status registers in pwrmgr_reg?

Reply:
yeah that's right, at the moment they are sync..what i'm saying is in the future there's no guarantee they will be.
Could you file an issue and track this for future release? Probably no need to tweak this now, but we should probably do so in the future.

Reply:
Sure, I will follow up with this update in another PR. The reason why I renamed clk_esc with clk_lc is to support err_q in LC domain in reg_top.sv.tpl for all IPs as follows. 

<%
  clk_lc_found = False;
  for clock in rb.clocks.values():
    if clock.clock == "clk_lc_i":
      clk_lc_found = True;
    endif
  endfor
%>\
% if clk_lc_found:
  always_ff @(posedge clk_lc_i or negedge rst_lc_ni) begin
    if (!rst_lc_ni) begin
% else:
  always_ff @(posedge ${reg_clk_expr} or negedge ${reg_rst_expr}) begin
    if (!${reg_rst_expr}) begin

I will need to tweak this script in another PR, too.

Reply:
this sounds good.
I just mean in the future, i think we will both clk_lc and clk_esc.
The logic (escalation receiver) that is using clk_esc then should be assumed to be async from clk_lc.

However since that is not the behvaior of this chip, and likely will not change in the near future, no reason to deal with it as long as we have a tracking issue for the future.

Reply:
Ah I see. Thanks for info. 
I filed https://github.com/lowRISC/opentitan/issues/14785 for future tracking.